app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -170,7 +171,7 @@ class EnabledContent extends ImmutableComponent {
   render () {
     const ledgerData = this.props.ledgerData
     const walletStatusText = walletStatus(ledgerData)
-    const inTransition = ledgerData.getIn(['migration', 'btc2BatTransitionPending']) === true
+    const inTransition = migrationState.inTransition(ledgerData)
```

Comment:
simple cleanup to put in helper method

Reply:
HOWEVER... notice that it says `migration` instead of `migrations`.  This was a bug I caught with the unit tests. I'm not sure how this ever returned true?

Reply:
This is failing for me and only see disabled screen on payments when upgrading from 0.18.x. This is the error:

```
(BLESSED_EXTENSION context for mnojpmjdmbbfmejpflffifhffcmidifd) http://localhost:8080/gen/aboutPages.entry.js:17915: Uncaught AssertionError: state must contain an Immutable.Map of migrations{AssertionError: state must contain an Immutable.Map of migrations
    at validateState (http://localhost:8080/gen/aboutPages.entry.js:173611:10)
    at Object.inTransition (http://localhost:8080/gen/aboutPages.entry.js:173626:13)
    at EnabledContent.render (http://localhost:8080/gen/aboutPages.entry.js:102839:41)
    at http://localhost:8080/gen/aboutPages.entry.js:78075:21
    at measureLifeCyclePerf (http://localhost:8080/gen/aboutPages.entry.js:77355:12)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (http://localhost:8080/gen/aboutPages.entry.js:78074:25)
    at ReactCompositeComponentWrapper._renderValidatedComponent (http://localhost:8080/gen/aboutPages.entry.js:78101:32)
    at ReactCompositeComponentWrapper.performInitialMount (http://localhost:8080/gen/aboutPages.entry.js:77641:30)
    at ReactCompositeComponentWrapper.mountComponent (http://localhost:8080/gen/aboutPages.entry.js:77537:21)
    at Object.mountComponent (http://localhost:8080/gen/aboutPages.entry.js:25250:35)}
```

Reply:
This is working because this is not `state` but `ledgerData` which is generated here https://github.com/brave/browser-laptop/blob/f52a10989bb9c632c2fbbfecb2ec538fa28cfc7c/app/browser/tabs.js#L243. We can't use state helper here

Reply:
OK I can fix this- we are OK to use the state helper because the path is identical, but it might be sloppy to do so (in case things change). The problem in our case is the validation that is done, which throws the error.

I'll change it back to the hardcoded path since the state helper shouldn't be used to evaluate ledgerData (even if the path matches)

Reply:
wait- nevermind, I see the problem... fixing